### PR TITLE
Compatibility overhaul and Travis integration

### PR DESCRIPTION
**Pull request by midnightlynx**
Python 2.6, 2.7, 3.3 and 3.4 now supported.

### CODE REVIEW

File: duo_client/client.py [hunk 1]
```diff
@@ -1,7 +1,10 @@
 """
 Low level functions for generating Duo Web API calls and parsing results.
 """
-__version__ = '2.1'
+from __future__ import absolute_import
+from __future__ import print_function
+import six
```
Comment (mschwager) [inline duo_client/client.py]: Looks like we're importing `six` twice.

**Comment by midnightlynx** [comment]
Oops. Yes, we were. Fixed.

File: duo_client/client.py (hunk 1 above)
Comment (midnightlynx) [inline duo_client/client.py]: Oops. We were. Fixed now.

**Comment by mschwager** [comment]
Thanks again @midnightlynx!

**Pull request merged**